### DISCUSSION TITLE
is there a way to customize page style when export to pdf ?

### DISCUSSION THREAD

**Discussion by eggcaker** [fonsp/Pluto.jl#1548] [Questions about using Pluto]
I plan to using Pluto to write some API  document  and then export to pdf to my customer, my question  is  how can I customize the pdf style ,like document width, table style etc.  at least exclude some code block when export to pdf but keep in notebook ?

**Comment by ctrekker** [comment]
I would suggest editing the Pluto notebook's style with a cell containing
```julia
html"""
<style>
    /* your css style here */
</style>
"""
```

For example I could make the gap between cells much larger with:
```julia
html"""
<style>
pluto-cell {
    margin-bottom: 100px;
}
</style>
"""
```

There's also a package [PlutoPDF.jl](https://github.com/JuliaPluto/PlutoPDF.jl) that will let you export Pluto notebooks to PDFs from the command line.

Regarding conditionally showing cells when a notebook is rendered to a PDF, I'm not sure if that's possible at the moment, since rendering to a PDF and rendering to the browser are done the same way.

**Reply by eggcaker** [reply]
I tried change some css in notebook , it's works in notebook web html , but when export to pdf it's still small than web
